app.rb

```diff
@@ -0,0 +1,55 @@
+require 'sinatra'
+require 'sinatra/reloader'
+require "bundler/setup"
+require "pg"
+require "active_record"
+require "pry"
+require_relative "db/connection"
+require_relative "models/apartment"
+require_relative "models/tenant"
+require_relative "Controllers/apartments_controller.rb"
+require_relative "Controllers/tenants_controller.rb"
+
+
+get "/" do
+  erb :home
+end
+
+# The homepage should list several menu options:
+# List all apartments (a link to GET /apartments)
+
+
+# View an apartment's details(a link to GET /apartments/1)
+get "/apartments/:id" do
```

Comment:
seems you have some duplicate routes, these routes here and below are very similar to the ones defined in your controllers.